### PR TITLE
add:models and endpoint

### PR DESCRIPTION
**Pull request by spectroalias**
please review, this is just the draft 1.

### CODE REVIEW

File: push/urls.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from .views import subscribe_client
+
+
+app_name = 'push'
+
+urlpatterns = {
+    path('subscribe/', subscribe_client, name="subscribe"),
+}
```
Comment (tucosaurus) [inline push/urls.py]: Add new line

File: push/urls.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from .views import subscribe_client
+
+
+app_name = 'push'
+
+urlpatterns = {
+    path('subscribe/', subscribe_client, name="subscribe"),
```
Comment (tucosaurus) [inline push/urls.py:8]: api names are nouns and not verbs/actions
would suggest `subscriptions`

File: push/views.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+from django.http import JsonResponse
+# Create your views here.
+
+
+@api_view(['POST'])
+@permission_classes([AllowAny])
+def subscribe_client(request, *args, **kwargs):
+    # api logic to save the data and make client a subscriber goes here
+    message = "Test"
+    return JsonResponse({"response": message})
```
Comment (tucosaurus) [inline push/views.py:12]: Use rest framework's response class here

File: templates/index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
```
Comment (tucosaurus) [inline templates/index.html]: delete this file

**Review by tucosaurus — APPROVED** [review]

File: push/views.py (hunk 3 above)
Comment (spectroalias) [reply push/views.py:12]: will change in the latest branch to avoid conflicts.

**Pull request merged**